src/components/origami.js

```diff
@@ -15,19 +16,24 @@ scene.add(directionalLight);
 
 const raycaster = new THREE.Raycaster();
 const mouse = new THREE.Vector2();
+
 let closestVertexIndex = -1;
+let areMarkersAtSamePosition = false;
 
-const createPointsMarker = () => {
+const createPointsMarker = color => {
   const geometry = new THREE.SphereGeometry(0.03, 16, 16);
-  const material = new THREE.MeshBasicMaterial({ color: '#098CEA' });
+  const material = new THREE.MeshBasicMaterial({ color });
   const marker = new THREE.Mesh(geometry, material);
   marker.visible = false;
   return marker;
 };
 
-const pointsMarker = createPointsMarker();
+const pointsMarker = createPointsMarker('#098CEA');
 scene.add(pointsMarker);
 
+const redMarker = createPointsMarker('#FF0000');
```

Comment:
재사용성을 고려해 색상을 따로 상수화 파일에서 관리하는 것은 어떨까요?
```suggestion
// constants.js 파일
const POINTS_MARKER_COLOR = '#098CEA';
const RED_MARKER_COLOR = '#FF0000';

//사용할 파일
const pointsMarker = createPointsMarker(POINTS_MARKER_COLOR);
const redMarker = createPointsMarker(RED_MARKER_COLOR);
```

이 파일 말고도 paper.js 파일에도 상수화할 부분이 있을 것 같아요!

Reply:
상수화 해두면 좋을 것 같습니다!
paper.js랑 위 부분의 코드들 상수화해서 올려놓겠습니다!